lib/init.coffee

```diff
@@ -6,13 +6,13 @@ module.exports =
     executablePath:
       title: 'Executable Path'
       type: 'string'
-      default: 'pep257'
-      description: "Path to executable pep257 cmd."
+      default: 'pydocstyle'
```

Comment:
That's not a bad idea, although maybe it would be better to just alert users they need to update?